examples/response_stream/public/components/page.tsx

```diff
@@ -31,11 +30,7 @@ export const Page: React.FC<PageProps> = ({ title = 'Untitled', children }) => {
           </EuiTitle>
         </EuiPageHeaderSection>
       </EuiPageHeader>
-      <EuiPageContent>
-        <EuiPageContentBody style={{ maxWidth: 800, margin: '0 auto' }}>
-          {children}
-        </EuiPageContentBody>
-      </EuiPageContent>
+      <EuiPageSection style={{ maxWidth: 800, margin: '0 auto' }}>{children}</EuiPageSection>
```

Comment:
Can you pass a `css` prop instead? And can we avoid using a magic number? I wonder if there is a variable provided by EUI for page width. 

```suggestion
      <EuiPageSection css={{ maxWidth: 800, margin: '0 auto' }}>{children}</EuiPageSection>
```

Reply:
Used the opportunity and migrated to `EuiPageTemplate` instead. That got rid of that custom css altogether and fixes the padding problem on the whole page too!

Before:

<img width="1273" alt="image" src="https://github.com/elastic/kibana/assets/230104/6719f701-55fe-4df2-9900-a559f3210c90">

After:

<img width="1261" alt="image" src="https://github.com/elastic/kibana/assets/230104/674bf148-8a4a-4b63-90fe-60de1c0c22c7">